lib/ansible/modules/setup.py

```diff
@@ -154,6 +154,9 @@
 # Only collect the default minimum amount of facts:
 # ansible all -m ansible.builtin.setup -a 'gather_subset=!all'
 
+# Display only facts related to date and time.
+# ansible all -m ansible.builtin.setup -a 'filter=ansible_date_time*'
```

Comment:
There is already an example of using `filter` above, no need to add another one.